test/elixir_bench/benchmarks/benchmarks_test.exs

```diff
@@ -2,17 +2,16 @@ defmodule ElixirBench.BenchmarksTest do
   use ElixirBench.DataCase
   alias ElixirBench.{Benchmarks, Benchmarks.Job}
   alias ElixirBench.Repos
+  import ElixirBenchWeb.TestHelpers
```

Comment:
this uhhh... shouldn't work? Afaik/iirc (macros aren't my strong suite admittedly) - but iirc you'd need to `reuquire` a module to use its macros. I'm confused, CI passes :thinking: 

@michalmuskala ? :D

Reply:
`import` implicitly does a `require`. I'm not a huge fan of this behaviour, but it's probably far too late to change it now.

Reply:
:scream: :scream: :scream: :cry: :cry: :cry: 

Thanks Michał